app/src/main/java/br/org/cesar/knot_setup_app/activity/configureDevice/ConfigureDevicePresenter.java

```diff
@@ -156,13 +157,12 @@ public void onServiceDiscoveryComplete(){
             }
 
             @Override
-            public void onServiceDiscoveryFail(){
-                Log.d("DEV-LOG","Service discovery failed");
-            }
+            public void onServiceDiscoveryFail(){ LogWrapper
```

Comment:
I suggest you use the same method indentation as you were using on the previous commit.